native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -39,8 +40,17 @@ se::Object *jsMouseEventObj = nullptr;
 se::Object *jsKeyboardEventObj = nullptr;
 se::Object *jsControllerEventArray = nullptr;
 se::Object *jsResizeEventObj = nullptr;
-se::Object *jsOrientationEventObj = nullptr;
 bool inited = false;
+
+// attach the argument object to the function
+void accessCacheArgObj(se::Object *func, se::Value *argObj, const char *cacheKey = "__reusedArgumentObject") {
```

Comment:
the argument name of orientation is __reusedArgumentObject ?

Reply:
It is the field name assigned to the function.